apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/dokument/Model.kt

```diff
@@ -104,6 +99,21 @@ data class Journalpost(
     val datoOpprettet: String,
 )
 
+enum class Journalstatus {
```

Comment:
Denne er henta frå eit anna system, sant? Kan vera greitt med ein kommentar som seier kor lista er henta frå, så ikkje nokon startar å tenkje at "hm, denne statusen treng vi ikkje, ekspedert er ikkje relevant for oss, eg slettar".

Reply:
Godt poeng! Legger det til i #3870